tools/tpm2_rsadecrypt.c

```diff
@@ -211,12 +219,16 @@ ENTRY_POINT(rsadecrypt) {
             .sapi_context = sapi_context
     };
 
-    ctx.session_data.sessionHandle = TPM_RS_PW;
+        ctx.session_data.sessionHandle = TPM_RS_PW;
 
     bool result = init(argc, argv, &ctx);
     if (!result) {
         return 1;
     }
 
+    if (INPUT_SESSION_HANDLE) {
+        ctx.session_data.sessionHandle = 0x3000000;
```

Comment:
magic numbers need comments. Macro's are nice for these, as they increase readability.